app.js

```diff
@@ -9,12 +9,20 @@ app.get('/api/topics', getTopics)
 
 app.get('/api', listApis)
 
+app.get('/api/articles/:article_id/comments', listComments)
+
 app.all("*", (req, res) => {
     res.status(404).send({ msg: "path not found" });
 });
 
 app.use((err, req, res, next) => {
-    res.status(500).send({ msg: "internal server error" });
+    if (err.code === "22P02") {
```

Comment:
There is nothing wrong with this chain as it is, however you may find it harder to debug as this gets bigger. Consider the very bottom of the error handler notes. There is a section on how to make error handler functions for each particular use case.